app/src/main/java/org/oppia/app/player/exploration/ExplorationFragment.kt

```diff
@@ -14,6 +15,9 @@ class ExplorationFragment : InjectableFragment() {
 
   override fun onAttach(context: Context) {
     super.onAttach(context)
+    val storyTextSize =
+      arguments!!.getString(ExplorationActivity.EXPLORATION_ACTIVITY_STORY_DEFAULT_FONT_SIZE_ARGUMENT_KEY)
+    FontScaleConfigurationUtil(context, storyTextSize).adjustFontScale()
```

Comment:
The `storyTextSize ` is a `String?` and the value which is passed in `FontScaleConfigurationUtil ` is `String`. Therefore make sure that the `storyTextSize` is non-null first.